lua/neotest-vitest/init.lua

```diff
@@ -61,7 +61,21 @@ local function hasVitestDependency(path)
     return false
   end
 
-  return hasVitestDependencyInJson(packageJsonContent) or hasRootProjectVitestDependency()
+  local gitRootPath = util.find_git_ancestor(path)
+  local hasRootMonorepoVitestDependency = false
+
+  -- only check the git root's package.json if it's different (e.g. in monorepos)
+  if rootPath ~= gitRootPath then
+    local monorepoSuccess, monorepoRootPackageJsonContent =
+      pcall(lib.files.read, gitRootPath .. "/package.json")
```

Comment:
test are failing because gitRootPath can be `nil`

Reply:
woops, should be fixed now!